style.css

```diff
@@ -90,3 +90,356 @@
+  padding-bottom: 10%;
+}
+
+.about-me-intro {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background: #1c1a19;
+  padding: 120px 24px;
+  background-image: url(./images/illustration-about-me-mobile.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  padding-bottom: 40%;
+}
+
+.about-me-title {
+  width: 100%;
+  height: auto;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.about-me-details {
+  width: 100%;
+  padding-top: 5%;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.btn-resume {
+  align-items: center;
+  width: 156px;
+  height: 48px;
+  text-decoration: none;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  padding: 10px;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+  border: 1px solid #ff6b00;
+  margin-top: 25px;
+}
+
+#skills-separator {
+  width: 85%;
+  margin: 0 auto;
+  margin-top: 8%;
+  background: #fff;
+}
+
+.skills-list-container {
+  width: 100%;
+  padding-top: 27%;
+  height: auto;
+  background: #1c1a19;
+  padding-bottom: 10%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.skill-list-bock {
+  width: 100%;
+}
+
+.skills-list li {
+  text-align: center;
+  list-style-type: none;
+  width: 100%;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 194%;
+  align-items: center;
+  color: #fff;
+  margin-top: 17px;
+}
+
+.skill-title {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  color: #fff;
+  text-align: left;
+  text-align: center;
+  margin-top: 20px;
+  width: 100%;
+}
+
+#skill-title-1 {
+  font-family: "InaiMathi", sans-serif;
+}
+
+#skill-title-2,
+#skill-title-3 {
+  font-family: "Inter", sans-serif;
+}
+
+#rectangle-icon-1 {
+  transform: rotate(-45deg);
+  margin-right: 15px;
+}
+
+#rectangle-icon-2 {
+  margin-right: 15px;
+}
+
+#circle-icon {
+  margin-right: 15px;
+}
+
+.see-project-btn:active {
+  background-color: #ff3c00;
+}
+
+.btn-resume:active {
+  background-color: #ff3c00;
+}
+
+.project-btn:active {
+  background-color: rgb(126, 126, 126);
+}
+
+.btn-see-project:active {
+  background-color: #ff3c00;
+}
+
+.recent-button:active {
+  background-color: rgb(126, 126, 126);
+}
+
+/* a:active {
+  box-shadow: 7px 6px 28px 1px rgba(0, 0, 0, 0.24);
+  transform: translateY(4px);
+} */
+
+.see-project-btn:hover {
+  background-color: #d2ada2;
+  border: 1px solid #ff3c00;
+}
+
+.btn-resume:hover {
+  background-color: #d2ada2;
+  border: 1px solid #ff3c00;
+}
+
+.project-btn:hover {
+  background-color: rgba(220, 213, 213, 0.854);
+}
+
+.btn-see-project:hover {
+  background-color: #d2ada2;
+  border: 1px solid #ff3c00;
+}

```

Comment:
- [ ] Good job implementing the hover effect as suggested👍 However, the `hover` styling is looking a bit different from the one given in the design guidelines ⚠ In Figma template, the background color for `hover` interaction is `#FFF5E1` and border is `1px solid #F55800` Similarly, the color is `#F55800`, Kindly fix this ⛏✨

_Please make sure to apply this for all other buttons_

Your design | Figma design
---| ---|
![image](https://github.com/FilbertKimasa/Portfolio/assets/106119105/add4aa01-8f01-4b8e-a5c3-2436253c5619) | ![image](https://github.com/FilbertKimasa/Portfolio/assets/106119105/a1094a50-5382-40b8-b02b-bfe4f95b2621)